tests/conftest.py

```diff
@@ -1698,7 +1707,7 @@ def snapshot(snapshot: SnapshotAssertion) -> SnapshotAssertion:
 
 
 @pytest.fixture
-def disable_block_async_io() -> Generator[Any, Any, None]:
+def disable_block_async_io() -> Generator[None]:
```

Comment:
I spotted this small incorrect type hint when adjusting the above `service_calls` fixture.